resources/views/mail/welcome.blade.php

```diff
@@ -0,0 +1,8 @@
+<x-mail::message>
+# Welcome To LearnHub MK
+
+Thanks for your interest
```

Comment:
I'll think of some text here and we can update this part later.

Reply:
As an example I have added some custom text